index.js

```diff
@@ -2,32 +2,18 @@ var turfPointOnLine = require('turf-point-on-line');
 var turfDistance = require('turf-distance');
 var turfLineDistance = require('turf-line-distance');
 var turfLineSlice = require('turf-line-slice');
+
 module.exports = function(opts) {
     /**
     * Configuration options
-    * @param {object} `units` - either `miles` or `km`. `maxReRouteDistance` - max distance the user can be from the route. `maxSnapToLocation` - max distance to snap user to route.
+    * @param {object} `units` - either `miles` or `km`. `maxReRouteDistance` - max distance the user can be from the route. `maxSnapToLocation` - max distance to snap user to route. `completionDistance` - distance away from end of step that is considered a completion. If this distance is shorter than the step distance, it will be changed to 10ft. `warnUserTime` - number of seconds ahead of maneuver to warn user about maneuver.
     */
     var options = {
         units: opts.units || 'miles',
-        maxReRouteDistance: opts.maxReRouteDistance || 0.03,
-        maxSnapToLocation: opts.maxSnapToLocation || 0.01
-    };
-
-    /**
-     * Given a user location and route, calculates whether re-routing should occur.
-     * @param {object} user point feature representing user location. Must be a valid GeoJSON object.
-     * @param {object} route from [Mapbox directions API](https://www.mapbox.com/developers/api/directions/).
-     * The Mapbox directions API returns an object with up to 2 `routes` on the `route` key. `shouldReRoute` expects of these routes, either the first or second.
-     * @returns {boolean} should user be re-routed.
-     */
-    function shouldReRoute(user, route) {
-        var r = {
-            type: 'Feature',
-            properties: {},
-            geometry: route.geometry
-        };
-        var closestPoint = turfPointOnLine(r, user);
-        return turfDistance(user, closestPoint, options.units) > options.maxReRouteDistance ? true : false;
+        maxReRouteDistance: opts.maxReRouteDistance || 0.0284091, // 150 ft
```

Comment:
I'm going to do a bigger overhaul of this when I fix https://github.com/mapbox/navigation.js/issues/20